api/database/migrations/2022_12_09_230812_drop_language_ability_users.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Database\Helpers\ApiEnums;
+use Illuminate\Support\Facades\DB;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $users = DB::table('users')->get();
+        foreach($users as $user) {
+            $languageAbility = $user->language_ability;
+            $userId = $user->id;
+            // before dropping the field language_ability, execute a data change provided it is non-null
+            if ($languageAbility !== null) {
+
+                // only override looking_for_english if it is null
+                if ($user->looking_for_english == null) {
+                    // play it safe, ENGLISH and BILINGUAL map to true
+                    if ($languageAbility == ApiEnums::LANGUAGE_ABILITY_ENGLISH | $languageAbility == ApiEnums::LANGUAGE_ABILITY_BILINGUAL) {
+                        DB::statement(
+                            <<<SQL
+                                UPDATE users
+                                SET looking_for_english = true
+                                WHERE id = :userId
+                            SQL, [
+                                'userId' => $userId,
+                            ]
+                        );
```

Comment:
For simple statements like this I recommend using the [Laravel DB methods](https://laravel.com/docs/9.x/queries#update-statements) since they include built-in sanitization and stuff like that.
```suggestion
                        DB::table('users')->where('id', $userId)->update(['looking_for_english' => true]);
```

Reply:
so many ways to do things, neat, thanks
done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5101/commits/da8d2eda2ea250476cc76e23c404c91708c9759d

as well, noticed a bug that was resolved by https://github.com/GCTC-NTGC/gc-digital-talent/pull/5101/commits/6e8ad5cc315b12b9736fa310dd7d36937a008ce3

Reply:
per my latter mistake, an issue sorta relevant exists here #4222